.github/workflows/testBuild.yml

```
@@ -0,0 +1,182 @@
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: Expensify/App/.github/actions/composite/setupNode@main
+
+      - uses: ruby/setup-ruby@eae47962baca661befdfd24e4d6c34ade04858f7
+        with:
+          ruby-version: '2.7'
+          bundler-cache: true
+
+      - name: Install cocoapods
+        uses: nick-invision/retry@0711ba3d7808574133d713a0d92d2941be03a350
+        with:
+          timeout_minutes: 10
+          max_attempts: 5
+          command: cd ios && pod install
+
+      - name: Decrypt profile
+        run: cd ios && gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output chat_expensify_adhoc.mobileprovision.gpg chat_expensify_adhoc.mobileprovision.gpg.gpg
+        env:
+          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+      - name: Decrypt certificate
+        run: cd ios && gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output Certificates.p12 Certificates.p12.gpg
+        env:
+          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+      - name: Run Fastlane
+        run: bundle exec fastlane ios build_test
+        env:
+          S3_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          S3_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          S3_BUCKET: ad-hoc-expensify-cash
+          S3_REGION: us-east-1
+          PULL_REQUEST_NUMBER: ${{ github.event.number }}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ios
+          path: ./ios_paths.json
+
+  # web:
+  #   name: Build and deploy Web
+  #   needs: validateActor
+  #   if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+  #       with:
+  #         fetch-depth: 0
+  #         ref: ${{ github.event.pull_request.head.sha }}
+
+  #     - uses: Expensify/App/.github/actions/composite/setupNode@main
+
+  #     - name: Configure AWS Credentials
+  #       # Version: 1.5.5
+  #       uses: aws-actions/configure-aws-credentials@e97d7fbc8e0e5af69631c13daa0f4b5a8d88165b
+  #       with:
+  #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  #         aws-region: us-east-1
+
+  #     - name: Build web for staging
+  #       run: npm run build-staging
+
+  #     - name: Build docs
+  #       run: npm run storybook-build
+  #       continue-on-error: true
+
+  #     - name: Deploy to S3 for internal testing
+  #       run: aws s3 cp --recursive --acl public-read "$GITHUB_WORKSPACE"/dist s3://ad-hoc-expensify-cash/web/"$PULL_REQUEST_NUMBER"
+  #       env:
+  #         PULL_REQUEST_NUMBER: ${{ github.event.number }}
+
+  postGithubComment:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
```

Can we update this to something like:
```suggestion
    name: Post a GitHub comment with app download links for testing
```